.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest a new feature
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature request**
+A simple description of what you would like to be added to `cabal`.
```

Comment:
```suggestion
A simple description of what you would like to be added to Cabal.
```

I use `cabal` to denote the tool and "Cabal" to denote the project, which includes several things besides the tool. Otherwise, it sounds like additions to `Cabal` (the library) are not covered.